src/ld65/config.c

```diff
@@ -1848,6 +1851,29 @@ unsigned CfgProcess (void)
             /* Remember the start address before handling this segment */
             unsigned long StartAddr = Addr;
 
+            /* Take note of overlayed segments and make sure there are no other
+            ** segment types following them in current memory region.
+            */
+            if (S->Flags & SF_OVERLAY) {
+            {
+                if (S->Flags & (SF_OFFSET | SF_START)) {
+                    ++Overlays;
+                } else {
+                    CfgError (GetSourcePos (M->LI),
+                              "Segment `%s' of type `overlay' requires either"
+                              " `Start' or `Offset' argument to be specified.",
```

Comment:
No period at the end of the message string.

Reply:
Also, please say that they are an `attribute` instead of an `argument`.